apps/test/integration/levelSolutions/applab/ec_design.js

```diff
@@ -1187,9 +1187,10 @@ module.exports = {
         dragElement(button[0], 350, 0);
         assert.equal(designModeViz.find('#design_button1').length, 0, "button was deleted");
 
+        // TODO(dave): re-enable this after we move to headless chrome
```

Comment:
Trying this in headless chrome is a bit of a long shot. I wasn't able to reproduce the test failure on these lines locally or on adhoc, but since Chrome sometimes behaves differently it seems worth trying again if/when we move to headless chrome. I suppose I could confirm this by kicking off another headless chrome run on circle to see if this test passes or not, before putting this comment in.